ci/tasks/run-e2e.sh

```
@@ -35,24 +36,69 @@ time bosh -n ucc \
-time bosh -n run-errand -d e2e-test elb-registration-test
-time bosh -n run-errand -d e2e-test heavy-stemcell-test
-time bosh -n run-errand -d e2e-test encrypted-heavy-stemcell-test
+# time bosh -n run-errand -d e2e-test iam-instance-profile-test
+# time bosh -n run-errand -d e2e-test raw-ephemeral-disk-test
+# time bosh -n run-errand -d e2e-test elb-registration-test
+# time bosh -n run-errand -d e2e-test heavy-stemcell-test
+# time bosh -n run-errand -d e2e-test encrypted-heavy-stemcell-test
 
 # spot instances do not work in China
 region=$( jq -e --raw-output ".region" environment/metadata )
 if [[ "${region}" != "cn-north-1" ]]; then
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this if expression.

(SC1009)